metadata_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGithubRegexp(t *testing.T) {
+	cases := map[string]string{
+		`git https://github.com/paultag/go-config">`: "paultag/go-config",
+		`https://github.com/owner/repo->`:            "owner/repo-",
```

Comment:
(don't know why I can't reply the last commet, so reply here)
seems if we want to relax the pattern, more situations needed to consider.
I'm fine to give up using restricted regex pattern. Let's use HTML parser instead, which is not a tough task actually. I will implement this later :)